aggregation/cardinality/src/main/java/com/spotify/heroic/aggregation/cardinality/ReduceHyperLogLogPlusCardinalityBucket.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2019 Spotify AB.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.heroic.aggregation.cardinality
+
+import com.clearspring.analytics.stream.cardinality.CardinalityMergeException
+import com.clearspring.analytics.stream.cardinality.HyperLogLogPlus
+import com.spotify.heroic.metric.Metric
+import com.spotify.heroic.metric.Payload
+
+import java.io.IOException
+import java.util.concurrent.ConcurrentLinkedQueue
+
+/**
+ * Bucket that counts the number of seen events.
+ *
+ * @author udoprog
+ */
+data class ReduceHyperLogLogPlusCardinalityBucket(
```

Comment:
Not related to this PR but would be great to document what all these different aggregations do. and if they are not used/useful, remove them?